temporalcli/commandsmd/commands.md

```diff
@@ -490,67 +1188,109 @@ Includes options set for [overlap-policy](#options-set-for-overlap-policy).
+### temporal schedule delete: Delete a Schedule
 
-The `temporal schedule delete` command deletes a Schedule.
-Deleting a Schedule does not affect any Workflows started by the Schedule.
+Deletes a Schedule on the front end Service:
 
-If you do also want to cancel or terminate Workflows started by a Schedule, consider using `temporal
-workflow delete` with the `TemporalScheduledById` Search Attribute.
+```
+temporal schedule delete --schedule-id YourScheduleId
+``` 
 
+Removing a schedule won't affect the Workflow Executions it started that
+are still running. To cancel or terminate these Workflow Executions, use 
+`temporal workflow delete` with the `TemporalScheduledById` Search Attribute.
+ 
 #### Options
 
 Includes options set for [schedule-id](#options-set-for-schedule-id).
 
-### temporal schedule describe: Get Schedule configuration and current state.
+### temporal schedule describe: Get Schedule configuration and current state
 
-The `temporal schedule describe` command shows the current configuration of one Schedule,
-including information about past, current, and future Workflow Runs.
+Show a Schedule configuration including information about past, current, 
+and future Workflow runs:
+
+```
+temporal schedule describe --schedule-id YourScheduleId
+```
 
 #### Options
 
 Includes options set for [schedule-id](#options-set-for-schedule-id).
 
-### temporal schedule list: Lists Schedules.
+### temporal schedule list: List Schedules
+
+Lists the Schedules hosted by a Namespace:
 
-The `temporal schedule list` command lists all Schedules in a namespace.
+```
+temporal schedule list --namespace YourNamespace
+```
 
 #### Options
 
-* `--long`, `-l` (bool) - Include detailed information.
-* `--really-long` (bool) - Include even more detailed information that's not really usable in table form.
+* `--long`, `-l` (bool) -
+  Show detailed information
+* `--really-long` (bool) -
+  Show extensive information in non-table form.
 
-### temporal schedule toggle: Pauses or unpauses a Schedule.
+### temporal schedule toggle: Pause or unpause a Schedule
 
-The `temporal schedule toggle` command can pause and unpause a Schedule.
+Pause or unpause a Schedule:
 
-Toggling a Schedule takes a reason. The reason will be set as the `notes` field of the Schedule,
-to help with operations communication.
+```
+temporal schedule toggle \
+    --schedule-id "YourScheduleId" \
+    --pause \
+    --reason "YourReason"
+```
 
-Examples:
+and
+
+```
+temporal schedule toggle
+    --schedule-id "YourScheduleId" \
+    --unpause \
+    --reason "YourReason"
+```
 
-* `temporal schedule toggle --schedule-id 'your-schedule-id' --pause --reason "paused because the database is down"`
-* `temporal schedule toggle --schedule-id 'your-schedule-id' --unpause --reason "the database is back up"`
+The reason updates the Schedule's `notes` field to support operations
+communication. It defaults to "(no reason provided)" when omitted.
```

Comment:
```suggestion
--reason updates the Schedule's `notes` field to support operations
communication. It defaults to "(no reason provided)" when omitted.
```

Reply:
Updated. Can't wait until I can do the next pass, which should be much quicker.